server/data/data-cassandra/src/main/java/org/apache/james/rrt/cassandra/migration/MappingsSourcesMigration.java

```diff
@@ -58,7 +96,25 @@ public Result run() {
     private Mono<Result> migrate(Pair<MappingSource, Mapping> mappingEntry) {
         return cassandraMappingsSourcesDAO.addMapping(mappingEntry.getRight(), mappingEntry.getLeft())
             .map(any -> Result.COMPLETED)
-            .doOnError(e -> LOGGER.error("Error while performing migration of mappings sources", e))
+            .doOnSuccess(success -> additionalInformation.incrementSuccessfulCount())
+            .doOnError(e -> {
+                LOGGER.error("Error while performing migration of mappings sources", e);
```

Comment:
Could you log some extra information about `MappingSource` and `Mapping`? I think it would be helpful.